build.gradle

```diff
@@ -150,14 +150,19 @@ dependencies {
     implementation group: 'org.json', name: 'json', version: '20190722'
 
 	//https://mvnrepository.com/artifact/com.nimbusds/nimbus-jose-jwt
-    implementation group: 'com.nimbusds', name: 'nimbus-jose-jwt', version: '8.3'
+    implementation group: 'com.nimbusds', name: 'nimbus-jose-jwt', version: '9.31'
```

Comment:
please validate that it is backward compatible. Try to test other levels of JWT vulnerability,

Reply:
I have validated the other JWT vulnerabilities and they work correctly, ensuring that they stay within the expected parameters.

Reply:
Thank you !!! ❤️